testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -941,6 +941,14 @@
   row.set(true)
 end
 
+When(/^I uncheck "([^"]*)" in the list$/) do |text|
```

Comment:
I would say we have already a step definition for this purpose. (I'm on my mobile app, I can't check it)

Reply:
i tried the existing `uncheck` steps originally but they all could not find the button, which is why i copied and adjusted the `I check "([^"]*)" in the list` step